include/vctr/Containers/Span.h

```diff
@@ -189,7 +189,7 @@ class Span : public VctrBase<ElementType, std::span<ElementType, extent>, extent
     template <class DstType>
     constexpr auto castTo()
     {
-        using SizeRatio = std::ratio<sizeof (DstType) / sizeof (ElementType)>;
+        using SizeRatio = std::ratio<sizeof (ElementType), sizeof (DstType)>;
```

Comment:
Yep, this was double wrong. We did not only switch numerator and denominator but also didn't pass them correctly to `std::ratio` as comma seperated arguments but as a fraction. Since `std::ratio` has a default value of 1 defined for the denominator this lead to an integer division being performed and the result being applied as numerator. Took me quite some time to spot this...

Reply:
🙈